machine/v1alpha1/types_openstack.go

```diff
@@ -92,6 +92,12 @@ type OpenstackProviderSpec struct {
 	// The volume metadata to boot from
 	RootVolume *RootVolume `json:"rootVolume,omitempty"`
 
+	// AdditionalBlockDevices is a list of specifications for additional block devices to attach to the server instance
```

Comment:
Nit, should start with the json tag name for oc describe. Since this is a rawextension it doesn't actually have an effect though

Reply:
I'm doing it anyway; thanks for the suggestion.